app/views/backend/sales/_form.html.haml

```diff
@@ -13,6 +13,7 @@
   = f.referenced_association :responsible, source: {scope: :users}, new: {employed: 1}
   = f.referenced_association :transporter, source: :transporters
   = f.input :description
+  = f.association :creator, input_html: { value: current_user.id }, as: :hidden
```

Comment:
Risky. If we need this, it needs to be moved up into the controller.
Also: don't timestamps already handle this?